.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -8,11 +8,14 @@ about: Before you start, check whether a similar issue was already reported. Sea
 
 **Affected versions / Verwendete Versionen**
 
-REDAXO: z.B./e.g. 5.10
-PHP: z.B./e.g. 7.3.0
-Database: z.B./e.g. MySQL 5.7.0 or MariaDB 10.1.40
-Browser: z.B./e.g. Mozilla Firefox 70 for Windows
-AddOns: eine List von / a list of REDAXO-AddOns
+Insert a system report via `REDAXO > System > System report > As Markdown` or tell us more about the system:
```

Comment:
Vllt sogar mit kleinem screenshot ausm backend?

Ich denke ich würde keine alternative zum sysbericht zulassen und das bereitstellen des sysberichts noch mehr forcieren... die meisten leute wissen ohnehin nicht woher sie die einzelnen zahlen nehmen solllen

Reply:
> Vllt sogar mit kleinem screenshot ausm backend?

Man sieht den Text ja in dem Eingabefeld bei einem neuen Issue. Daher ist es mit einem Screenshot schwierig.

Ich würde den Text übrigens als HTML-Kommentar schreiben, sodass er nicht im Issue erscheint, wenn man ihn drin lässt.

Reply:
Auch eine schöne Idee @gharlan